app/scripts/TrackRenderer.js

```diff
@@ -1157,6 +1157,11 @@ class TrackRenderer extends React.Component {
           // some track types (like overlay-track don't have a track info)
           trackOrientation = TRACKS_INFO_BY_TYPE[trackDef.type].orientation;
         }
+
+        if (trackAtZoomStart.originalTrack) {
+          // this is a LeftTrackModifier track so it's vertical
```

Comment:
I don't understand how `trackAtZoomStart.originalTrack` is related to the track orientation, specifically why we want this track to be vertical. Do you mind adding a comment?

Reply:
I just changed the line to `if (trackAtZoomStart  instanceof LeftTrackModifier)`. Hopefully that's clearer.

Reply:
Oh yeah! Now it makes a lot of sense :)